Update log level to verbose when unrecognized status (#25095)
Azure/azure-sdk-for-js: sdk/core/core-lro/CHANGELOG.md

## 2.5.2 (Unreleased)

### Bugs Fixed

> - Change the LRO warning to verbose to reduce confusion when we meet unrecognized operation status

Reviewer: nitpick: no need to mention reasons in the changelog:
```suggestion
- Change the logging level for unrecognized operation statuses from warning to verbose.
```
